src/app/state/operations/utils.ts

```diff
@@ -0,0 +1,3 @@
+export type Callback<T> = (err?: Error, res?: T) => void
+
+export const callbackify = (promise: () => Promise<any>, cb: (arg0: undefined, arg1: undefined) => any) => cb ? promise().then((res: any) => cb(undefined, res)).catch((err: any) => cb(err, undefined)) : promise()
```

Comment:
I didn’t find any import for this

Reply:
Thanks, I tried using it and was unsuccessful, I'll remove it.